index.html

```diff
@@ -40,9 +44,179 @@ <h1 class="description">I am Desmond, glad to see you!
+                 <div class="card">
+                    <img class="card-image" src="./images/gymfit-snapshot.png" alt="gymfit-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Tonic</h1>
+                        <div class="skills">
+                            <span>CANOPY</span>
+                            <span class="dot"></span>
+                            <span>Back End Dev</span>
+                            <span class="dot"></span>
+                            <span>2015</span>
+                        </div>
+                        <div class="card-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </div>
+                        <div class="card-skills-btns-container">
+                            <button type="button" class="card-skills-btn">HTML</button>
+                            <button type="button" class="card-skills-btn">CSS</button>
+                            <button type="button" class="card-skills-btn">JavaScript</button>
+                        </div>
+                        <button type="button" class="see-projects-btn">See projects</button>
+                    </div>
+                 </div>
+                <!-- End of card 3-->
+
+                 <!-- Card 4 section-->
+                 <div class="card">
+                    <img class="card-image" src="./images/professional-art-snapshot.jpg" alt="professional-art-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Multi-Project Stories</h1>
+                        <div class="skills">
+                            <span>CANOPY</span>
+                            <span class="dot"></span>
+                            <span>Back End Dev</span>
+                            <span class="dot"></span>
+                            <span>2015</span>
+                        </div>
+                        <div class="card-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </div>
+                        <div class="card-skills-btns-container">
+                            <button type="button" class="card-skills-btn">HTML</button>
+                            <button type="button" class="card-skills-btn">CSS</button>
+                            <button type="button" class="card-skills-btn">JavaScript</button>
+                        </div>
+                        <button type="button" class="see-projects-btn">See projects</button>
+                    </div>
+                 </div>
+            </div>
+        </div>
+    </section>
+    <!-- End of work section-->
+
+    <!-- The about section -->
+
+    <section class="aboutMyself-container">
+        <div class="aboutMyself-content">
+            <h1 class="about-title">About Myself</h1>
```

Comment:
- [ ] It is good practice to have only one `<h1>` per page, and that should be for the main title. Kindly change this to any of h2 - h6 tag. You can read more [here](https://www.hobo-web.co.uk/headers/)

Reply:
- This issue is not yet resolved :x: